synapse/logging/opentracing.py

```diff
@@ -617,23 +617,25 @@ def set_operation_name(operation_name: str) -> None:
 
 
 @only_if_tracing
-def force_tracing(span=Sentinel) -> None:
+def force_tracing(span: "opentracing.Span" = Sentinel) -> None:  # type: ignore[assignment]
     """Force sampling for the active/given span and its children.
 
     Args:
         span: span to force tracing for. By default, the active span.
     """
     if span is Sentinel:
-        span = opentracing.tracer.active_span
-    if span is None:
+        span_to_trace = opentracing.tracer.active_span
+    else:
+        span_to_trace = span
+    if span_to_trace is None:
```

Comment:
Seems to imply that `span` can be `None`, which runs counter to the type hint? Do we ever pass a `None` in somewhere?

Reply:
I think passing `None` in is invalid, but `opentracing.tracer.active_span` can be `None`? Will double check.

Reply:
Makes sense, I misread the code.